test-suite/src/lib.rs

```diff
@@ -238,6 +238,10 @@ macro_rules! generate_transaction_alter_table_tests {
             };
         }
 
+        glue!(
+            alter_table_rename_table,
```

Comment:
This test is combination of two optional features `alter-table` and `transaction`.
So, it would be better to use `transaction_alter_table_rename_table` as a test name

Reply:
Thanks!